Sources/UIApplication+handleSDLEvents.swift

```diff
@@ -254,23 +254,27 @@ public func onNativeTouch(
     else { return }
 
     Task { @MainActor in
+        guard let screenScale = UIScreen.main?.scale else {
+            return
+        }
+
         var event = SDL_Event(tfinger:
-        SDL_TouchFingerEvent(
-            type: eventType.rawValue,
-            timestamp: UInt32(timestampMs),
-            touchId: Int64(touchDeviceId), // some arbitrary number, stays the same per device
-            fingerId: Int64(pointerFingerId),
-            x: x / Float(UIScreen.main.scale),
-            y: y / Float(UIScreen.main.scale),
-            dx: 0,
-            dy: 0,
-            pressure: pressure
+            SDL_TouchFingerEvent(
+                type: eventType.rawValue,
+                timestamp: UInt32(truncatingIfNeeded: timestampMs),
```

Comment:
I'm pretty sure we established this is not the cause of the crash already, but there is no harm in this change: if `timestampMs` fits inside `UInt32`, then it will remain unchanged, and if it doesn't, things will still work fine.

Reply:
Update: no, actually this is almost certainly the crash. 49 days of system uptime would lead to a UInt32 overflowing. It's not actually that unlikely that a device has been on for 49 days, especially newer, better devices. I will see if I can improve this further

Reply:
I further improved this by making `timestampMs` start from the time the SDLActivity was inited, rather than from when the device was started. This combined with `truncatingIfNeeded` should fix this problem permanently and without weirdness 🙏🏼